### PR TITLE
Update RIF 1.6.0

### PR DESCRIPTION
**Pull request by bnagirniak**
### PURPOSE
New RIF 1.6.0 is available.

### EFFECT OF CHANGE
Updating RIF to 1.6.0 version.

### TECHNICAL STEPS
Updated RIF submodule, create_sdk.py and API_VERSION calculation.

### CODE REVIEW

**Review by ardraeiss — APPROVED** [review]
Looks fine for me, waiting for QA approval

**Comment by bsavery** [comment]
Status of QA review @Dizzyspb ?

**Review by Dizzyspb — APPROVED** [review]
RIF integration is tested by QA. Approved for merge.
No new issues found.

**Pull request merged**
